### ISSUE TITLE
import error:  'Iterable' from 'collections' 

### ISSUE THREAD

**Issue by panzerballett** [massgh/ipyslides#4]
i tried to run ipyslides in python 3.10 and python 3.12 and both failed for the same reason:

```
import ipyslides as isd
...
ImportError: cannot import name 'Iterable' from 'collections' (/usr/lib/python3.12/collections/__init__.py)
```

this is for `ipyslides==4.3.7`.

`Iterable` is now  (starting from python 3.10) in `collections.abc`.

**Comment by asaboor-gh** [comment]
This is fixed now starting from version 4.4.0. Thanks for pointing out.

**Comment by panzerballett** [comment]
great! thanks a lot!
